openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -749,8 +753,8 @@ Authorization: BEARER czZCaGRSa3F0MzpnWDFmQmF0M2JW
    }
 }
 ```
-
-Below is a non-normative example of a Credential Request for a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format using a Credential instance identifier and key proof type `jwt`:
+Note: Either proof or proofs parameter is REQUIRED if the proof_types_supported parameter is non-empty and present in the credential_configurations_supported parameter of the Issuer metadata for the requested Credential.
```

Comment:
```suggestion

Either `proof` or `proofs` parameter MUST be present if the `proof_types_supported` parameter is present in the `credential_configurations_supported` parameter of the Issuer metadata for the requested Credential.

```

usually we would put "This parameter MUST be present if the `proof_types_supported` parameter is present in the `credential_configurations_supported` parameter of the Issuer metadata for the requested Credential" in the definitions of both `proof` and `proofs` parameters above.

also empty `proof_types_supported` parameter would be illegal, no?